src/library_glfw.js

```diff
@@ -780,7 +780,7 @@ var LibraryGLFW = {
 
         var filename = allocateUTF8(path);
         filenamesArray.push(filename);
-        setValue(filenames + i*4, filename, 'i8*');
+        setValue(filenames + i*4, filename, '{{{ POINTER_TYPE }}}');
```

Comment:
I suppose one advantage of `i8*` or `void*` would be that you don't need the `{{{ }}}` here. Not sure how often that happens though, it looks like this is the only place in this PR.

Reply:
We should remove the final uses of setValue.. so it should no longer exist: See #16849

Reply:
LGTM. I guess it doesn't matter much then. POINTER_TYPE seems more obvious just from reading than `void*` so this seems fine as it is.